### PR TITLE
Console bridge logging

### PR DESCRIPTION
**Pull request by reinzor**
Follow up of https://github.com/ros-drivers/odva_ethernetip/pull/14

Addresses #13 

Changed CI to industrial CI so we can build on kinetic.

### CODE REVIEW

**Review by mikepurvis — APPROVED** [review]
Looks great, I dig it!

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,18 @@
-sudo: required
-dist: trusty
-language: generic
+sudo: required 
+dist: trusty 
+language: generic
```
Comment (mikepurvis) [inline .travis.yml]: Avoid these trailing spaces, please.

File: src/session.cpp [hunk 2]
```diff
@@ -341,26 +333,25 @@ void Session::closeConnection(size_t n)
...
+    CONSOLE_BRIDGE_logWarn("IO packet received with %d bytes, but only %d bytes used", n, reader.getByteCount());
   }
 
   return result;
 }
 
 void Session::sendIOPacket(CPFPacket& pkt)
 {
-  // cout << "Sending CPF Packet on IO Socket" << endl;
+  // CONSOLE_BRIDGE_logInform("Sending CPF Packet on IO Socket");
```
Comment (mikepurvis) [inline src/session.cpp]: Consider changing these to `logDebug` and uncommenting?

Note as well that when rosconsole interprets the log lines from console_bridge, I believe it treats anything before a colon as if it was the tag for the purposes of ROS_XXXX_NAMED. So if you want the option to filter these commented-out lines separately when turning the log level to debug, you could give them such a prefix.

See implementation here: https://github.com/ros/rosconsole_bridge/blob/kinetic-devel/src/bridge.cpp#L52-L56

Comment (reinzor) [reply src/session.cpp]: Thanks for the comment, didn't know that. I uncommented the log msgs and changed the inform for debug.

**Pull request merged**